core/src/main/java/org/hisp/dhis/android/core/user/internal/LogInCall.kt

```diff
@@ -84,15 +76,15 @@ internal class LogInCall @Inject internal constructor(
         val parsedServerUrl = ServerUrlParser.parse(trimmedServerUrl)
         ServerURLWrapper.setServerUrl(parsedServerUrl.toString())
 
-        val authenticateCall = userService.authenticate(
-            UserIdAuthenticatorHelper.basic(username!!, password!!),
-            UserFields.allFieldsWithoutOrgUnit(null),
-        )
-
-        val credentials = Credentials(username, trimmedServerUrl!!, password, null)
+        val credentials = Credentials(username!!, trimmedServerUrl!!, password, null)
 
         return try {
-            val user = apiCallExecutor.executeObjectCallWithErrorCatcher(authenticateCall, apiCallErrorCatcher)
+            val user = coroutineAPICallExecutor.wrap {
```

Comment:
The `apiCallErrorCatcher` is removed. It must be passed as a parameter to the `wrap` function.